packages/grafana-ui/src/components/EmptyState/GrotNotFound/GrotNotFound.tsx

```diff
@@ -24,20 +24,23 @@ export const GrotNotFound = ({ width = 'auto', height }: Props) => {
 
   useEffect(() => {
     const handleMouseMove = (event: MouseEvent) => {
-      const grotArm = svgRef.current?.querySelector('#grot-not-found-arm');
-      const grotMagnifier = svgRef.current?.querySelector('#grot-not-found-magnifier');
+      // only apply animation if no reduced motion preference is set
+      if (window.matchMedia('(prefers-reduced-motion: no-preference)').matches) {
```

Comment:
```suggestion
      if (!window.matchMedia('(prefers-reduced-motion: no-preference)').matches) {
        return;
```
What about doing an early return instead? ;)

Reply:
sure 👍